modules/osd-create-cluster-ccs-gcp.adoc

```diff
@@ -0,0 +1,267 @@
+// Module included in the following assemblies:
```

Comment:
I don't see this module in the preview, but also it shouldn't be there anyway? Confused as to why it's here now.